docs/index.md

```diff
@@ -1,17 +1,17 @@
 ---
 layout: page
-title: AddressBook Level-3
+title: SETA
 ---
 
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
```

Comment:
This is pointing to the AB3 repo if I am not wrong, the link in the README.md can be used here too!

Reply:
done!